cpu/stm32/periph/rtc_mem.c

```diff
@@ -133,7 +138,12 @@ void rtc_mem_read(unsigned offset, void *data, size_t len)
 
     uint8_t *out = (uint8_t *)data;
 
+#if defined(CPU_FAM_STM32U5)
```

Comment:
```suggestion
#if defined(TAMP_BKP0R)
```

should make this a bit more future-proof